scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -2191,13 +2272,25 @@ static bool layered_monitor(void)
 	return true;
 }
 
+static bool timer_antistall_reset(void){
```

Comment:
`set` and `reset` usually indicating turning a bit to 1 and 0 respectively which makes this naming rather confusing.